agent/agent.go

```diff
@@ -3644,6 +3643,7 @@ func serviceMaintCheckID(serviceID structs.ServiceID) structs.CheckID {
 // EnableServiceMaintenance will register a false health check against the given
 // service ID with critical status. This will exclude the service from queries.
 func (a *Agent) EnableServiceMaintenance(serviceID structs.ServiceID, reason, token string) error {
+	a.logger.Printf("Putting service %s into maintenance", serviceID.String())
```

Comment:
Should this have a log level and pkg prefix? Like:

```suggestion
	a.logger.Printf("[INFO] agent: Putting service %s into maintenance", serviceID.String())
```

Reply:
It doesn't really need to be there at all. Was just helpful when debugging. But apparently it looked real enough I didn't remove it before.